src/visualstates/gui/dialogs/librariesdialog.py

```diff
@@ -81,12 +81,15 @@ def deleteButtonClicked(self):
         self.librariesChanged.emit(self.libraries)
 
     def addClicked(self):
-        self.libraries.append(self.libraryNameEdit.text())
+        input = self.libraryNameEdit.text().strip()
+        for library in self.libraries:
```

Comment:
Instead of this loop you can directly check whether the text in the list as `if input in self.libraries`

Reply:
Sorry I forgot that was possible in python. Will change it.